Templates/Per Tenant Extension/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -254,6 +254,25 @@ jobs:
         with:
           shell: ${{ inputs.shell }}
           project: ${{ inputs.project }}
+          testsToAnalyze: "app"
+
+      - name: Analyze BCPT Test Results
+        id: analyzeTestResultsBCPT
+        if: (success() || failure()) && env.doNotRunBcptTests == 'False'
```

Comment:
Is this env variable set?

I believe it needs to be included in "Read Settings", line 97.

Same for `doNotRunpageScriptingTests`.